www/new-prebid-limes.uk/default.html

```diff
@@ -0,0 +1,378 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="icon" type="image/svg+xml" href="noun_Swan_3263882.svg">
+  <title>New Prebid Limes | SWAN Demo</title>
+  <link href="bootstrap.min.css" rel="stylesheet">
+  <link href="blog.css" rel="stylesheet">
+  <link href="advert.css" rel="stylesheet">
+
+  <script>
+    var FAILSAFE_TIMEOUT = 10000;
+
+    window.swan_offers = {
+        advert1: JSON.parse({{ .NewOfferIDAsString }})
+    };
+
+    window.adInfo = function(offer) {
+      var owid = "http://cmp.swan-demo.uk/info?owid=" + btoa(window.swan_offers[offer]);
+      window.open(owid, "_blank");
+    };
+
+    var adUnits = [
+        {
+            code: 'test-div',
+            mediaTypes: {
+                banner: {
+                    sizes: [[300,250]]
+                }
+            },
+            bids: [
+                {
+                    bidder: 'openx',
+                    params: {
+                        delDomain: 'sademo-d.openx.net',
+                        unit: '540949377'
+                    }
+                },
+                {
```

Comment:
here are 2 swan bidders which both are in the network and receive the swan_offer_name for each auction